xml/api/crm_mon-2.10.rng

```diff
@@ -20,7 +20,7 @@
             <ref name="node-attributes-list" />
         </optional>
         <optional>
-            <ref name="node-history-list" />
+            <externalRef href="node-history-2.10.rng" />
```

Comment:
At some point we're going to have to figure out how to automatically regenerate files that reference another file.

E.g. if A-2.10.rng references B, and B-2.11 is created, then some make target should generate an A-2.11 that's identical to A-2.10 but references B-2.11 (or if A-2.11 already exists, just seds the B version in it).

Reply:
Yeah, I have been thinking we need that sort of thing too.  That's gonna be some real makefile fun.